hasura.planx.uk/migrations/1708016067772_run_sql_migration/up.sql

```diff
@@ -0,0 +1,49 @@
+DROP VIEW public.analytics_summary;
```

Comment:
I was surprised to see a migration here - but it looks to be the same migration from #2798, right? If you change your base branch to Daf's branch it may display a more meaningful comparison of changes, then if #2798 merges first it should automatically switch to `main` again & rebase ?

Reply:
I created my branch off of Daf's but when I created the PR I didn't notice I was pointing at `main` I'll get that sorted 👍